### ISSUE TITLE
alternative name for "play area"? 

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/natural-selection#27]
What name do we want to use for referring to this part of the UI?  This is important because it will appear in the element id in the PhET-iO API.

<img width="476" alt="screenshot_19" src="https://user-images.githubusercontent.com/3046552/70561574-ee212900-1b47-11ea-8988-38a7d240ce9c.png">

We've been using the term "play area".  But "play area" is being used by a11y.  Every screen now has a [PlayAreaNode](https://github.com/phetsims/scenery-phet/blob/master/js/accessibility/nodes/PlayAreaNode.js), with its own very specific meaning, and adding another `playAreaNode` would be both confusing and problematic.  So we'll need to either come up with a different name, or revisit the a11y decision to adopt this term.

The currently implementation is [ViewportNode](https://github.com/phetsims/natural-selection/blob/master/js/common/view/ViewportNode.js).
It's element id is `naturalSelection.labScreen.view.viewportNode`.

**Comment by amanda-phet** [comment]
I would cringe every time I called that the "play area" because it doesn't seem appropriate to me either. I would be happy with "displayArea" or "viewportNode".

**Comment by ariel-phet** [comment]
Since this is specific to the natural selection sim, I would go with something like `environmentDisplayNode` or something like that, make it very clear we are talking about the spot where environments and bunnies are shown.

**Comment by kathy-phet** [comment]
I like the direction @ariel-phet is going since this is Natural Selection specific. Other thoughts: 
ecosystemDisplayNode
ecosystemAnimationNode

**Comment by pixelzoom** [comment]
To clarify... I'm not suggesting that "play area" is an appropriate term, and in fact I feel that it's inappropriate.  But I've heard "play area" used in design discussion more than once, and I want to make sure that we choose a term that's appropriate to be "public facing" in the PhET-iO API.

**Comment by pixelzoom** [comment]
Re ecosystem vs environment...  https://difference.guru/difference-between-an-environment-and-an-ecosystem/

> An **environment** refers to the surroundings or dwelling place of all living things while an **ecosystem** is likened to a community that functions as a single unit. When you talk about the environment, it does not include the relationship between organisms and their surroundings, but only the setting or habitat.

Places that we're currently using "environment" (which will appear in the PhET-iO API):

EnvironmentRadioButtonGroup - for selecting equator vs arctic
EnvironmentalFactorsPanel - for selecting wolves, tough food, limited food
EnvironmentModel - part of the model that deals with bunnies and environmental factors
EnvironmentalFactor - model base class for wolves, tough food, limited food

**Comment by pixelzoom** [comment]
Re ecosystem vs environment...  https://difference.guru/difference-between-an-environment-and-an-ecosystem/

> An **environment** refers to the surroundings or dwelling place of all living things while an **ecosystem** is likened to a community that functions as a single unit. When you talk about the environment, it does not include the relationship between organisms and their surroundings, but only the setting or habitat.

Places that we're currently using "environment" (which will appear in the PhET-iO API):

model:
EnvironmentModel - sub-model that deals with bunnies and environmental factors
EnvironmentalFactor - base class for wolves, tough food, limited food
Environments - enumeration of "EQUATOR" and "ARCTIC"

view:
EnvironmentRadioButtonGroup - for selecting equator vs arctic
EnvironmentalFactorsPanel - for selecting wolves, tough food, limited food
EnvironmentNode - node that switches background image between equator and arctic

Perhaps we need to rename some of these too?

**Comment by pixelzoom** [comment]
12/11/19 design meeting @amanda-phet @kathy-phet @ariel-phet @pixelzoom 

Let's use `EnvironmentDisplayNode`.   
Example element id will be `naturalSelection.labScreen.view.environmentDisplayNode`.

We won't change any of the other names listed in https://github.com/phetsims/natural-selection/issues/27#issuecomment-564702077.

**Comment by pixelzoom** [comment]
Done, renamed as follows:

ViewportNode -> EnvironmentDisplayNode
EnvironmentNode -> EnvironmentBackgroundNode

**Comment by pixelzoom** [comment]
Reopening this issue to be revisited.

We decided on  `EnvironmentDisplayNode` for the Node being considered in this issue.  While that name violates well-established PhET naming conventions, I was willing to go along with it.  But then #33 came along and proposed to give sub-models names like "populationDisplayModel".  So I see a pattern here that I need to push back on.

PhET has a convention of model and view elements of a Model-View Controller relationship. The convention is to name a model class with the noun that describe it (e.g. `Particle`) and the view components with the same noun and a "Node" suffix (e.g. `ParticleNode`).  The "Node" suffix indicates that it's specific to the view.  "DisplayNode" is like saying "ViewNode", which is redundant.  "DisplayModel" is like saying "ViewModel", which is confusing. 

The model class in this sim is `Environment` -- it includes all of the model elements that exist in the environment, and the interactions between those model elements.  It is therefore appropriate to call the Node that displays it `EnvironmentNode`.   So I've changed to that name. 

In PhET-iO design, we will encounter tensions between what you might like to see in the Studio tree, and well-establish naming conventions in code.  In those cases, it's my recommendation that that coding conventions need to take precedence.
